src/clj/athens/self_hosted/clients.clj

```diff
@@ -40,6 +40,7 @@
 
 
 ;; Public send API
+;; TODO: validate send messages from source
```

Comment:
Currently client and server only do validation when they receive events. They should validate when they send as well. This is the unhelpful error that is thrown when an event is sent from server that does not much schema.

```
Mon Jun 21 13:07:33 EDT 2021 [worker-4] ERROR - handle websocket frame org.httpkit.server.Frame$TextFrame@30ea0ede
java.lang.RuntimeException: java.lang.Exception: Not supported: class org.httpkit.server.AsyncChannel
        at com.cognitect.transit.impl.WriterFactory$1.write(WriterFactory.java:65)
        at cognitect.transit$write.invokeStatic(transit.clj:167)
        at cognitect.transit$write.invoke(transit.clj:164)
        at athens.self_hosted.clients$__GT_transit.invokeStatic(clients.clj:31)
        at athens.self_hosted.clients$__GT_transit.invoke(clients.clj:27)
        at athens.self_hosted.clients$send_BANG_.invokeStatic(clients.clj:47)
        at athens.self_hosted.clients$send_BANG_.invoke(clients.clj:43)
        at athens.self_hosted.web.presence$hello_handler.invokeStatic(presence.clj:16)
        at athens.self_hosted.web.presence$hello_handler.invoke(presence.clj:33)
        at athens.self_hosted.web.presence$presence_handler.invokeStatic(presence.clj:77)
        at athens.self_hosted.web.presence$presence_handler.invoke(presence.clj:74)
        at athens.self_hosted.components.web$make_receive_handler$receive_handler__45504.invoke(web.clj:55)
        at clojure.core$partial$fn__5857.invoke(core.clj:2628)
        at org.httpkit.server.AsyncChannel.messageReceived(AsyncChannel.java:153)
        at org.httpkit.server.WSHandler.run(RingHandler.java:183)
        at org.httpkit.server.LinkingRunnable.run(RingHandler.java:150)
        at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
        at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.Exception: Not supported: class org.httpkit.server.AsyncChannel
        at com.cognitect.transit.impl.AbstractEmitter.marshal(AbstractEmitter.java:194)
        at com.cognitect.transit.impl.AbstractEmitter.emitArray(AbstractEmitter.java:97)
        at com.cognitect.transit.impl.AbstractEmitter.marshal(AbstractEmitter.java:182)
        at com.cognitect.transit.impl.JsonEmitter.emitMap(JsonEmitter.java:171)
        at com.cognitect.transit.impl.AbstractEmitter.emitMap(AbstractEmitter.java:85)
        at com.cognitect.transit.impl.AbstractEmitter.marshal(AbstractEmitter.java:184)
        at com.cognitect.transit.impl.AbstractEmitter.emitArray(AbstractEmitter.java:97)
        at com.cognitect.transit.impl.AbstractEmitter.marshal(AbstractEmitter.java:182)
        at com.cognitect.transit.impl.JsonEmitter.emitMap(JsonEmitter.java:171)
        at com.cognitect.transit.impl.AbstractEmitter.emitMap(AbstractEmitter.java:85)
        at com.cognitect.transit.impl.AbstractEmitter.marshal(AbstractEmitter.java:184)
        at com.cognitect.transit.impl.AbstractEmitter.marshalTop(AbstractEmitter.java:211)
        at com.cognitect.transit.impl.JsonEmitter.emit(JsonEmitter.java:41)
        at com.cognitect.transit.impl.WriterFactory$1.write(WriterFactory.java:62)
        ... 20 more
```

@neotyk